apps/advcasio/metadata.json

```diff
@@ -18,8 +18,5 @@
   "storage": [
 			{"name":"advcasio.app.js","url":"app.js"},
 			{"name":"advcasio.img","url":"app-icon.js","evaluate":true}
-  ],
-	"data": [
-			{ "name": "advcasio.data.json", "url": "data.json", "storageFile": true }
```

Comment:
@gfwilliams is there any way to clean up possible leftover data files on people's watches when they upgrade?

Reply:
Not as far as I'm aware, no. I guess we could add something but in the scheme of things I guess one or two files kicking around isn't the end of the world.

Reply:
That's fair, thanks!